app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -22,6 +22,8 @@ CONFIG_POWER_DOMAIN_INTEL_ADSP=y
 CONFIG_ADSP_IMR_CONTEXT_SAVE=y
 CONFIG_ADSP_IDLE_CLOCK_GATING=y
 
+CONFIG_ARCH_CPU_IDLE_CUSTOM=y
```

Comment:
Btw, why do you need this? I don't see a custom idle handled added in this patch. 

This setting is now enabled in upstream by default (the new icache-workaround), so no need to enable this on SOF side anymore.